mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -5851,6 +5851,7 @@ func TestAccContainerCluster_WithCPAFeatures(t *testing.T) {
 
 	// Bootstrap KMS keys and needed IAM role.
 	diskKey := acctest.BootstrapKMSKeyWithPurposeInLocationAndName(t, "ENCRYPT_DECRYPT", "us-central1", "control-plane-disk-encryption")
+	diskKey2 := acctest.BootstrapKMSKeyWithPurposeInLocationAndName(t, "ASYMMETRIC_DECRYPT", "us-central1", "control-plane-disk-encryption2")
```

Comment:
The update test step is continuing to fail with the same error, but I see a couple of reasons that might be happening. The first is that `diskKey2` needs a purpose of `ENCRYPT_DECRYPT` to create a symmetric key.